### DISCUSSION TITLE
Line Layer has capRounded property working?

### DISCUSSION THREAD

**Discussion by miltonarce** [visgl/deck.gl#9223] [General]
when I tried to use capRounded with LineLayer didn't work. And I can't use PathLayer because LineLayer change the width of my route dinamycally.

I have to do this with PathLayer but it doen'st look good:
  getWidth: () => {
      let lineWidth = 15000;

      switch (true) {
        case zoom >= 16:
          lineWidth = 10;
          break;
        case zoom >= 15:
          lineWidth = 25;
          break;
        case zoom >= 14:
          lineWidth = 40;
          break;
        case zoom >= 13:
          lineWidth = 100;
          break;
        case zoom >= 12:
          lineWidth = 200;
          break;
        case zoom >= 11:
          lineWidth = 250;
          break;
        case zoom >= 10:
          lineWidth = 500;
          break;
        case zoom >= 9:
          lineWidth = 900;
          break;
        case zoom >= 8:
          lineWidth = 2000;
          break;
        case zoom >= 7:
          lineWidth = 4000;
          break;
        case zoom >= 6:
          lineWidth = 10000;
          break;
        default:
          lineWidth = 15000;
      }

      return lineWidth;
    },

It's an option add capRounded property in LineLayer or Do I have another way to handle lineWidth better  with PathLayer?

**Comment by Pessimistress** [comment]
> I can't use PathLayer because LineLayer change the width of my route dynamically

Not sure what you mean. LineLayer and PathLayer handle widths the same way, just with different [widthUnits](https://deck.gl/docs/api-reference/layers/path-layer#widthunits) defaults.
